spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -81,147 +71,586 @@ public class StreamApplicationTests {
-relationship with each other.
+This can mean, for example, dynamically choosing the destinations that these channels connect to at runtime (e.g. Kafka topics or Rabbit MQ exchanges), via external configuration properties in any form that is supported by Spring Boot (application arguments, environment variables, `application.yml` files, etc). Taking the source example from the previous section, providing the `spring.cloud.stream.bindings.output.destination=raw-sensor-data` property to the application will cause it to write to the `raw-sensor-data` Kafka topic, or the `orders` exchange in Rabbit.
 
-Spring Cloud Stream models this behavior through the concept of a consumer group, which is similar to the notion of
-consumer groups in Kafka. Each consumer binding can specify a group name such as
-`spring.cloud.stream.bindings.input.group=foo` (the actual name of the binding may vary). Each consumer group bound to
-a given destination will receive a copy of the published data, but within the group, only one application will receive
-each specific message.
+On the other hand, since the application is not middleware-specific, Spring Cloud Stream will automatically detect and use a binder that is found on the classpath, so you can easily use different types of middleware with the same code, just by including a different binder at build time. For more complex use cases, Spring Cloud Stream also provides the ability of packaging multiple binders within the same application and choosing what type of binder should be used at runtime, and even if multiple binders should be used at runtime for different channels.
 
-If no consumer group is specified for a given binding, then the binding is treated as if belonging to an anonymous,
-independent, single-member consumer group. Otherwise said, if no consumer group is specified for a binding, it will be
-in a publish-subscribe relationship with any other consumer groups.
+=== Persistent publish subscribe and consumer groups
 
-In general, it is preferable to always specify a consumer group when binding an application to a given destination.
-When scaling up a Spring Cloud Stream application, a consumer group must be specified for each of its input bindings,
-in order to prevent its instances from receiving duplicate messages (unless that behavior is desired, which is a less
-common use case).
+Spring Cloud Stream is a library focusing on building message-driven microservices, and more specifically stream processing applications. In such scenarios, communication between different logical applications follows a publish-subscribe pattern, with data being broadcast through shared topics. This can be seen in the following picture, which shows a typical deployment for a set of interacting Spring Cloud Stream applications.
 
-NOTE: This feature has been introduced since version 1.0.0.M4.
+.Spring Cloud Stream Application topologies
```

Comment:
The text following this visual is all about `SCSt-sensors.png` - wrong image linked, maybe?